docs/install/yarn_install.md

```diff
@@ -0,0 +1,236 @@
+
+```bash
+hadoop version
+Hadoop 2.7.1.2.3.1.0-2574
+Subversion git@github.com:hortonworks/hadoop.git -r f66cf95e2e9367a74b0ec88b2df33458b6cff2d0
+Compiled by jenkins on 2015-07-25T22:36Z
+Compiled with protoc 2.5.0
+From source with checksum 54f9bbb4492f92975e84e390599b881d
+This command was run using /usr/hdp/2.3.1.0-2574/hadoop/lib/hadoop-common-2.7.1.2.3.1.0-2574.jar
+```
+
+#### Spark
+Zeppelin can work with multiple versions Spark. A complete list [is available here.](https://github.com/apache/incubator-zeppelin#build) This document assumes Spark 1.3.1 is installed on Zeppelin node at /home/zeppelin/prerequisites/spark.
+
+## Build
+
+Checkout source code from [https://github.com/apache/incubator-zeppelin](https://github.com/apache/incubator-zeppelin)
+```bash
+cd /home/zeppelin/
+git clone https://github.com/apache/incubator-zeppelin.git
+```
+Zeppelin package is available at /home/zeppelin/incubator-zeppelin after the checkout completes.
+
+### Cluster mode
+
+As its assumed Hadoop 2.7.x is installed on the YARN cluster & Spark 1.3.1 is installed on Zeppelin node. Hence appropriate options are chosen to build Zeppelin. This is very important as Zeppelin will bundle corresponding Hadoop & Spark libraries and they must match the ones present on YARN cluster & Zeppelin Spark installation. 
+
+Zeppelin is a maven project and hence must be built with Apache Maven.
+
+```bash
+cd /home/zeppelin/incubator-zeppelin
+mvn clean package -Pspark-1.3 -Dspark.version=1.3.1 -Dhadoop.version=2.7.0 -Phadoop-2.6 -Pyarn -DskipTests
+```
+Building Zeppelin for first time downloads various dependencies and hence takes few minutes to complete. 
+
+## Zeppelin Configuration
+Zeppelin configurations needs to be modified to connect to YARN cluster. Create a copy of zeppelin environment XML
+```bash
+cp /home/zeppelin/incubator-zeppelin/conf/zeppelin-env.sh.template /home/zeppelin/incubator-zeppelin/conf/zeppelin-env.sh 
+```
+Set the following properties
+```bash
+export JAVA_HOME=/home/zeppelin/prerequisites/jdk1.7.0_79
+export HADOOP_CONF_DIR=/etc/hadoop/conf
+export ZEPPELIN_PORT=10008
+export ZEPPELIN_JAVA_OPTS="-Dhdp.version=2.3.1.0-2574"
+```
+As /etc/hadoop/conf contains various configurations of YARN cluster, Zeppelin can now submit Spark/Hive jobs on YARN cluster form its web interface. The value of hdp.version is set to 2.3.1.0-2574. This can be obtained by running the following command
+```bash
+hdp-select status hadoop-client | sed 's/hadoop-client - \(.*\)/\1/'
+# It returned  2.3.1.0-2574
+```
+## Start/Stop
+### Start Zeppelin
+
+```
+cd /home/zeppelin/incubator-zeppelin
+bin/zeppelin-daemon.sh start
+```
+After successful start, visit http://[zeppelin-server-host-name]:8080 with your web browser.
+Note that port **8081** also need to be accessible for websocket connection.
```

Comment:
Do i remove the live ?

On Sat, Aug 8, 2015 at 10:17 PM, Lee moon soo <notifications@github.com>
wrote:

> In docs/install/yarn_install.md
> <https://github.com/apache/incubator-zeppelin/pull/192#discussion_r36584593>
> :
>
> > +export ZEPPELIN_JAVA_OPTS="-Dhdp.version=2.3.1.0-2574"
> > +```
> > +As /etc/hadoop/conf contains various configurations of YARN cluster, Zeppelin can now submit Spark/Hive jobs on YARN cluster form its web interface. The value of hdp.version is set to 2.3.1.0-2574. This can be obtained by running the following command
> > +```bash
> > +hdp-select status hadoop-client | sed 's/hadoop-client - \(.*\)/\1/'
> > +# It returned  2.3.1.0-2574
> > +```
> > +## Start/Stop
> > +### Start Zeppelin
> > +
> > +```
> > +cd /home/zeppelin/incubator-zeppelin
> > +bin/zeppelin-daemon.sh start
> > +```
> > +After successful start, visit http://[zeppelin-server-host-name]:8080 with your web browser.
> > +Note that port **8081** also need to be accessible for websocket connection.
>
> Current master branch does not require separate websocket port anymore.
> While this document instruct building Zeppelin from master, I think this
> line won't be necessary. (branch-0.5 and release 0.5-incubating still
> requires separate websocket port)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-zeppelin/pull/192/files#r36584593>.
>



-- 
Deepak